packages/cactus-cmd-api-server/src/test/typescript/benchmark/artillery-api-benchmark.test.ts

```diff
@@ -143,7 +143,7 @@ test("Start API server, and run Artillery benchmark test.", async (t: Test) => {
   } else {
     const avgLatency = report.aggregate.latency.median;
     log.info(`Average Latency: ${avgLatency}`);
-    t.assert(avgLatency < 5, "Average latency should be less than 10");
+    t.assert(avgLatency < 1, "Average latency should be less than 1");
```

Comment:
@Leeyoungone My bad from earlier, the durations are specified / can be updated here, not inside the test case
https://github.com/hyperledger/cactus/blob/2599c00af68503094f18e6825f937eec3fc69f07/packages/cactus-cmd-api-server/src/test/resources/artillery/benchmark-script.yaml

Reply:
Thank you so much, fixed it!